housekeeper/store/api/core.py

```diff
@@ -7,19 +7,21 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker, scoped_session
 
+from housekeeper.store.api.handlers.update import UpdateHandler
 from housekeeper.store.models import Model
 from housekeeper.store.api.handlers.create import CreateHandler
 from housekeeper.store.api.handlers.read import ReadHandler
 
 LOG = logging.getLogger(__name__)
 
 
-class CoreHandler(ReadHandler, CreateHandler):
+class CoreHandler(CreateHandler, ReadHandler, UpdateHandler):
     """Aggregating class for the store api handlers"""
 
     def __init__(self, session):
         ReadHandler(session=session)
         CreateHandler(session=session)
+        UpdateHandler(session=session)
```

Comment:
This is not a pattern that we want to use for session management, just as a side note.